communication/README.md

```diff
@@ -78,7 +81,7 @@ please propose a specific date on the [Kubernetes Community Meeting Agenda].
 
 ## Conferences
 
-Kubernetes is the main focus of CloudNativeCon/KubeCon, held every spring in Europe and winter in North America. Information about these and other community events is available on the CNCF [events] pages.
+Kubernetes is the main focus of KubeCon + CloudNativeCon, held every spring in Europe and winter in North America. Information about these and other community events is available on the CNCF [events] pages.
```

Comment:
We should also add China here.

Reply:
Good idea! I just added it. Unfortunately, I also managed to accidentally add a merge commit when I was syncing my fork. As you can see, I also tried to revert it, which I think made a mess of things. I'd appreciate any suggestions for fixing this?